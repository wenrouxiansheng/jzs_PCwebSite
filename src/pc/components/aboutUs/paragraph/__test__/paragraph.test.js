import React from "react";
import renderer from "react-test-renderer";
import Paragraph from "../index";

describe("PC:<Paragraph />", () => {
  it("render", () => {
    const wrapper = renderer.create(
      <Paragraph text="测试" style={{ color: "red" }} />
    );
    let tree = wrapper.toJSON();

    //渲染测试
    expect(tree).toMatchSnapshot();
  });
});
