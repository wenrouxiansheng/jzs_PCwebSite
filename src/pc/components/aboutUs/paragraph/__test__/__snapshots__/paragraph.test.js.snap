// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PC:<Paragraph /> render 1`] = `
<div
  className="paragraph"
  dangerouslySetInnerHTML={
    Object {
      "__html": "测试",
    }
  }
  style={
    Object {
      "color": "red",
    }
  }
/>
`;
